charts/psmdb-operator/templates/deployment.yaml

```diff
@@ -33,7 +33,7 @@ spec:
           - percona-server-mongodb-operator
           env:
             - name: WATCH_NAMESPACE
-              value: {{ default .Release.Namespace .Values.watchNamespace }}
+              value: {{ if kindIs "invalid" .Values.watchNamespace }}{{ .Release.Namespace }}{{ else }}{{ .Values.watchNamespace | quote }}{{ end }}
```

Comment:
The trick (hack?) is needed because, contrary the PXC operator chart, no `watchAllNamespaces` property exists.
There was a proposal in https://github.com/percona/percona-helm-charts/pull/125 to use the same structure for the PSMDB operator.